prisma/schema.prisma

```diff
@@ -29,9 +18,8 @@ model Parent {
   province          province
   preferredLanguage locale            @map("preferred_language")
   createdAt         DateTime          @default(now()) @map("created_at") @db.Timestamptz(6)
-  updatedAt         DateTime          @default(now()) @map("updated_at") @db.Timestamptz(6)
+  updatedAt         DateTime?         @map("updated_at") @db.Timestamptz(6)
```

Comment:
why do we feel like this field cant be defaulted to the current timestamp

Reply:
I think there isn't really a need to set it to the default time since it's storing the same duplicate information as `createdAt`.

Reply:
Also, it doesn't really make sure for the updatedAt time to be defaulted because it's technically the time of creation rather than the time it was updated.